src/app/renderer/components/app/FileBrowser/FileBrowser.tsx

```diff
@@ -34,15 +34,16 @@ export class FileBrowser extends React.Component<Props, {}> {
     render() {
         return (
             <div className={styles.wrap}>
-                <Head projectName={this.props.project.config.name}/>
+                <Head projectName={this.props.project.config.name} version={this.props.appVersion}>
+                    <title>
```

Comment:
Head does not accept any children

Reply:
Yes seems you are right and yet my WebStorm tells me otherwise. :D

![Screenshot 2021-12-13 at 10 13 45](https://user-images.githubusercontent.com/2048953/145792912-5a499ce2-4f0b-4f2a-bc47-d8f7d3a56e32.png)